tests/unit/test_pandas_df_utils.py

```diff
@@ -110,19 +109,20 @@ def test_csv_to_libffm():
     import tempfile
     import os
 
-    filedir = tempfile.tempdir
+    filedir = tempfile.gettempdir()
```

Comment:
What is the difference? I'm using tempdir in other parts of the code

Reply:
This is an interesting thing - using `-s` in `pytest` is not compatible with `tempdir`. Did you notice anything similar?

Reply:
as I said to @loomlike, here maybe we could leverage the fixtures for temporary paths of pytest: https://docs.pytest.org/en/latest/tmpdir.html

Reply:
This is a good idea, as it will be used across different test suites. Has this been implemented in the data loader utility PR? I did not find it...